ambry-clustermap/src/main/java/com.github.ambry.clustermap/AmbryStateModelFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.clustermap;
+
+import org.apache.helix.participant.statemachine.StateModel;
+import org.apache.helix.participant.statemachine.StateModelFactory;
+
+
+class AmbryStateModelFactory extends StateModelFactory<StateModel> {
+  private final String zkAddr;
+
+  AmbryStateModelFactory(String zkAddr) {
+    this.zkAddr = zkAddr;
```

Comment:
There are `zkAddr` and `zkConnectStr` (in other classes). Are they the same thing?